client/data/authorizations/reducer.ts

```diff
@@ -18,10 +21,10 @@ const receiveAuthorizations = (
 		data = [],
 		error,
 	}: {
-		type: string;
-		query: Query;
-		data: Authorization[];
-		error: string;
+		type?: string;
```

Comment:
Hmm interesting. At first glance, it looks like this will change the signature of the reducer (ie moving away from `reducer( state, { type, error, data } )`) but I'll give it a fresh look tomorrow just to be sure :)

Reply:
Thank you! Fixed [b652639](https://github.com/Automattic/woocommerce-payments/pull/4949/commits/b65263975e9abcfba30aad048fa278d423107636)